coda.io.subdomain.json

```diff
@@ -0,0 +1,25 @@
+{  
+    "providerId":"coda.io",
+    "providerName":"Coda",
+    "serviceId":"subdomain",
+    "serviceName":"Coda Site",
+    "version": 1,
+    "syncPubKeyDomain" : "domainconnect.coda.io",
```

Comment:
One last change I mentioned before, this template requires `"hostRequired": true`